content/en/docs/upgrade/Air-Gapped-Upgrade-with-KubeKey.md

```diff
@@ -0,0 +1,236 @@
+---
+title: "Air-Gapped Upgrade with KubeKey"
+keywords: "Air-Gapped, kubernetes, upgrade, kubesphere, v3.0.0"
+description: "Air-Gapped Upgrade Kubernetes and KubeSphere"
+
+linkTitle: "Air-Gapped Upgrade with KubeKey"
+weight: 4020
+---
+Air-gapped upgrade with KubeKey is recommended for users whose KubeSphere and Kubernetes were both deployed by [All-in-One Installation](https://v2-1.docs.kubesphere.io/docs/installation/all-in-one/) or [Multi-node Installation](https://v2-1.docs.kubesphere.io/docs/installation/multi-node/). If your Kubernetes cluster was provisioned by yourself or cloud providers, please refer to [Air-Gapped Upgrade with ks-installer](../air-gapped-upgrade-with-ks-installer/).
+
+## Prerequisites
+
+- You need to have a KubeSphere cluster running version 2.1.1.
+
+{{< notice warning >}}
+
+If your KubeSphere version is v2.1.0 or earlier, please upgrade to v2.1.1 first.
+
+{{</ notice >}}
+
+- Download KubeKey and upload to the kubernetes server.
+```bash
+# md5: 212091024dca02f8f34323e00f4be81d
+wget https://kubesphere-installer.pek3b.qingstor.com/offline/v3.0.0/kubesphere-all-v3.0.0-offline-linux-amd64.tar.gz
+```
+- Docker Registry
+
+You need to have a harbor or other Docker registry.
+
+{{< notice tip >}}
+
+You can [quick start yourself docker registry](https://kubesphere.com.cn/forum/d/2240-docker-registry)
```

Comment:
This is a Chinese article.

Reply:
I will change this to official documents.